packages/pricing/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@medusajs/pricing",
+  "version": "0.0.1",
+  "description": "Medusa Pricing module",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "bin": {
+    "medusa-pricing-migrations-down": "dist/scripts/bin/run-migration-down.js",
+    "medusa-pricing-migrations-up": "dist/scripts/bin/run-migration-up.js",
+    "medusa-pricing-seed": "dist/scripts/bin/run-seed.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/medusajs/medusa",
+    "directory": "packages/pricing"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "author": "Medusa",
+  "license": "MIT",
+  "scripts": {
+    "watch": "tsc --build --watch",
+    "watch:test": "tsc --build tsconfig.spec.json --watch",
+    "prepublishOnly": "cross-env NODE_ENV=production tsc --build && tsc-alias -p tsconfig.json",
+    "build": "rimraf dist && tsc --build && tsc-alias -p tsconfig.json",
+    "test": "jest --runInBand --bail --forceExit -- src/**/__tests__/**/*.ts",
+    "test:integration": "jest --runInBand --forceExit -- integration-tests/**/__tests__/**/*.ts",
+    "migration:generate": " MIKRO_ORM_CLI=./mikro-orm.config.dev.ts mikro-orm migration:generate",
+    "migration:initial": " MIKRO_ORM_CLI=./mikro-orm.config.dev.ts mikro-orm migration:create --initial",
+    "migration:create": " MIKRO_ORM_CLI=./mikro-orm.config.dev.ts mikro-orm migration:create",
+    "migration:up": " MIKRO_ORM_CLI=./mikro-orm.config.dev.ts mikro-orm migration:up",
+    "orm:cache:clear": " MIKRO_ORM_CLI=./mikro-orm.config.dev.ts mikro-orm cache:clear"
+  },
+  "devDependencies": {
+    "@mikro-orm/cli": "5.7.12",
+    "cross-env": "^5.2.1",
+    "faker": "^6.6.6",
+    "jest": "^25.5.4",
+    "medusa-test-utils": "^1.1.40",
+    "pg-god": "^1.0.12",
+    "rimraf": "^3.0.2",
+    "ts-jest": "^25.5.1",
+    "ts-node": "^10.9.1",
+    "tsc-alias": "^1.8.6",
+    "typescript": "^4.4.4"
+  },
+  "dependencies": {
+    "@medusajs/modules-sdk": "^1.9.2",
+    "@medusajs/types": "^1.10.2",
```

Comment:
This was added to deps because of webpack complaints in Nextjs functions for product, same will apply here. 

I'll create a ticket to see what needs to be done to remove this dep.

Reply:
Got it, thx!

Reply:
Probably because of some `enum`s in there that will have to be moved.